caproto/_log.py

```diff
@@ -143,9 +149,133 @@ def color_logs(color):
 
 
 logger = logging.getLogger('caproto')
+ch_logger = logging.getLogger('caproto.ch')
+search_logger = logging.getLogger('caproto.bcast.search')
 current_handler = None  # overwritten below
 
 
+class CaprotoAdapter(logging.LoggerAdapter):
```

Comment:
Do we need this class anymore?

Reply:
No, I left it there temp in case we need for test. will clean it later

Reply:
Sensible.